e2e/manual-e2e.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { NestFactory } from '@nestjs/core';
+import { ApplicationModule } from './src/app.module';
+import { DocumentBuilder, SwaggerModule } from '../lib';
+import {
+  FastifyAdapter,
+  NestFastifyApplication
+} from '@nestjs/platform-fastify';
+import { INestApplication, Logger } from '@nestjs/common';
+import { ExpressAdapter } from '@nestjs/platform-express';
+
+const port = 4001;
+const host = 'localhost';
+const docRelPath = '/api-docs';
+
+async function bootstrap() {
+  const app = await NestFactory.create<NestFastifyApplication>(
+    ApplicationModule,
+    new FastifyAdapter()
+    // new ExpressAdapter()
```

Comment:
reminder to remove this comment

Reply:
manual-e2e has a capability to quickly switch between Fastify & Express by commenting/uncommenting Adapter

that was the initial idea
Do u think it's not ok?

Reply:
I think it's only a nit to be honest, but I personally dislike commented code if there's an alternative, for instance, using constants to control this flow. For instance:
```ts
   const EXP = 'express'
   const FST = 'fastify'
   const getAdapter = {
      express: () => new ExpressAdapter,
      fastify: () => new FastifyAdapter
    }


const app = await NestFactory.create(ApplicationModule, getAdapter[EXP]())   
  
```